querybook/server/lib/ai_assistant/tools/table_schema.py

```diff
@@ -255,7 +273,7 @@ def get_table_schema_by_name(
 
 def get_slimmed_table_schemas(
     table_schemas: list[dict], column_keys_to_keep=["name", "type"]
-) -> list[dict]:
+) -> list[TableSchema]:
```

Comment:
typing doesn't look right, also can you update the caller typing for table_schemas